src/PropertiesPanelController.cpp

```
@@ -0,0 +1,497 @@
+QColor PropertiesPanelController::panelColor() const
+{
+    return QApplication::palette().color(QPalette::Window);
+}
+
+QColor PropertiesPanelController::headerColor() const
+{
+    return QApplication::palette().color(QPalette::Window).darker(110);
+}
+
+QColor PropertiesPanelController::textColor() const
+{
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)